protocol/0010-MARG-margin_orchestration.md

```diff
@@ -11,6 +11,15 @@
 1. [ ] The mark price changes causing the trader’s margin to move into the close-out zone. A collateral search is initiated and the margin is topped back up to the search zone. No further actions are taken. (<a name="0010-MARG-009" href="#0010-MARG-009">0010-MARG-009</a>)
 1. [ ] The mark price changes causing the trader’s margin to move into the close-out zone. A collateral search is initiated and the margin is topped back up to a level which results in the trader still being in the close-out zone.  (<a name="0010-MARG-010" href="#0010-MARG-010">0010-MARG-010</a>)
 1. [ ] The mark price changes causing the trader’s margin to move in to the release level. Margin should be released back to the trader. (<a name="0010-MARG-011" href="#0010-MARG-011">0010-MARG-011</a>) 
+1. [ ] Maintenance margin is correctly calculated for a party with orders / positions when obtaining risk factors from the lognormal risk model under various different parameters: In particular we see that 
+    - maintenance margin obtained when sigma is 1.0 is lower than maintenance margin obtained when sigma is  (<a name="0010-MARG-012" href="#0010-MARG-012">0010-MARG-012</a>) 
+    - maintenance margin obtained when tau is 1.0/365.25/24 is lower than maintenance margin obtained when sigma is 1.0/365.25/24/12 (<a name="0010-MARG-013" href="#0010-MARG-013">0010-MARG-013</a>)
```

Comment:
you meant margin when tau is 1.0/365.25/24 comparing with "tau" is 1.0/365.25/24/12?